src/server/api/controllers/events.controller.js

```diff
@@ -0,0 +1,29 @@
+const knex = require('../../config/db');
+const Error = require('../lib/utils/http-error');
+
+const getEvents = async () => {
+  try {
+    return await knex('events').select('events.id', 'event.event_type');
```

Comment:
Please also return date, venue, description, and max_participants.

Reply:
added all these column names